test/tests/vectorpostprocessors/work_balance/tests

```diff
@@ -0,0 +1,20 @@
+[Tests]
+  [./test]
+    type = 'CSVDiff'
+    input = 'work_balance.i'
+    csvdiff = 'work_balance_out_work_balance_0000.csv'
+    min_parallel = 2
+    max_parallel = 2
+    mesh_mode = replicated
+  [../]
+
+  [./distributed]
+    type = 'CSVDiff'
+    input = 'work_balance.i'
+    cli_args = 'Outputs/file_base=distributed'
+    csvdiff = 'distributed_work_balance_0000.csv'
+    min_parallel = 2
+    max_parallel = 2
+    mesh_mode = distributed
```

Comment:
This shouldn't be necessary for a framework test, but I'll leave that up to you. We do run distributed mesh in parallel. This is fine.

Reply:
Oh right, good point. In fact there's a possibly it won't be stable on different platforms. I guess we'll see.